ocp-setup/rb-tower-container-group.yml

```diff
@@ -0,0 +1,14 @@
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: tower-container-group
+  namespace: tower
```

Comment:
hardcoded namespace

Reply:
Currently this is just part of manual documentation. 
Should be template in future. I want to talk about it on todays sync.